rest/model/distro.go

```diff
@@ -354,6 +354,7 @@ type APIDistro struct {
 	DisableShallowClone   bool                     `json:"disable_shallow_clone"`
 	UseLegacyAgent        bool                     `json:"use_legacy_agent"`
 	Note                  APIString                `json:"note"`
+	MountScript           APIString                `json:"mount_script"`
```

Comment:
mount command, also I think this will break in the rebase because of apistring

Reply:
this no longer applies (the variable was removed)